src/System.IO.Ports/pkg/System.IO.Ports.pkgproj

```diff
@@ -11,8 +11,5 @@
     <InboxOnTargetFramework Include="$(UAPvNextTFM)" /> 
     <ProjectReference Include="..\src\System.IO.Ports.csproj" />
   </ItemGroup>
-  <ItemGroup>
-    <InboxOnTargetFramework Include="$(UAPvNextTFM)" />
```

Comment:
Why was this removed?

Reply:
Because it was duplicated, it is right above before the project reference. It was just a cleanup